realm/realm-library/src/main/java/io/realm/RealmQuery.java

```diff
@@ -2322,7 +2322,7 @@ private OsResults lazyFindAll() {
 
     /**
      * Create a text-based predicate using the Realm Query Language. This predicate can be combined
-     * with other raw or type safe predicates.
+     * with other raw or type safe predicates, it accepts Realm values as arguments.
```

Comment:
"Realm values" meaning both Realm primitives and Realm Models? If so, I would explicitly write both down.